### PR TITLE
SRA ToolKit manual

### PR DESCRIPTION
**Pull request by 223050025**
Add manual to SRA ToolKit and put the manual in subfolder of NGS.

### CODE REVIEW

**Comment by 223050025** [comment]
@Milokita

File: NGS/SRA-ToolKit/pic/term.txt
Comment (HoshinoKyouna) [inline NGS/SRA-ToolKit/pic/term.txt:1]: What is this text file used for?

File: NGS/SRA-ToolKit/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+source ~/.bashrc
+```
+
+## Using the package
+
+Try to use SRA ToolKit to download a fastq files (convert from .sra) as example. 
+GSE71378's raw file id is SRR061633, use commands will split two original paired-end reads (paired sequencing sequences) into two files, with the first and second sequences of each paired reads stored separately
+
+```
+fastq-dump --split-files SRR061633
```
Comment (HoshinoKyouna) [inline NGS/SRA-ToolKit/README.md:44]: Some users recommended to use `prefetch` to download entire SRA files first before using `fastq-dump` to convert SRA files into raw fastq files.

**Review by HoshinoKyouna — APPROVED** [review]

File: NGS/SRA-ToolKit/README.md (hunk 1 above)
Comment (223050025) [reply NGS/SRA-ToolKit/README.md:44]: Thank you for your opinion, I think it is OK, choose different functions according to the specific needs of the project, the complete SRA file takes up more space, directly download fastq file takes up less space, and it is faster.

File: NGS/SRA-ToolKit/pic/term.txt
Comment (223050025) [reply NGS/SRA-ToolKit/pic/term.txt:1]: This is just a test to create a blank folder that I will delete later.

**Pull request merged**